sqldelight-runtime/src/main/java/com/squareup/sqldelight/db/SqlDatabaseHelper.kt

```diff
@@ -13,17 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.squareup.sqldelight;
+package com.squareup.sqldelight.db
 
-import android.database.Cursor;
-import android.support.annotation.CheckResult;
-import android.support.annotation.NonNull;
+interface SqlDatabaseHelper {
+  fun getConnection(): SqlDatabaseConnection
+  fun close()
 
-/** Creates instances of {@code T} from rows in a {@link Cursor}. */
-public interface RowMapper<T> {
-  /**
-   * Return an instance of {@code T} corresponding to the values of the current positioned row of
-   * {@code cursor}.
-   */
-  @CheckResult @NonNull T map(@NonNull Cursor cursor);
+  interface Callback {
+    fun onCreate(connection: SqlDatabaseConnection)
+    fun onUpgrade(connection: SqlDatabaseConnection, oldVersion: Int, newVersion: Int)
```

Comment:
this is a schema change?

Reply:
I'm going to remove this callback api for now. Its not obvious when they'd actually be needed and I think as long as we provide the `onCreate` and `onUpgrade` the person using SqlDelight can call into them whenever the driver they're using gives them a callback.